src/SDKs/HDInsight/Management.HDInsight.Tests/UnitTests/CustomizationTests.cs

```diff
@@ -72,6 +72,7 @@ public void TestEnableHttpCustomization()
             Assert.Equal(handler.Requests[0], handler.Requests[1]);
         }
 
+        [Fact(Skip ="Enabled to appease warnings, needs fixing")]
```

Comment:
in cases where we cant mark these as internal methods, report the projects to @shahabhijeet